src/packtPublishingFreeEbook.py

```diff
@@ -201,32 +201,69 @@ def __init__(self, currentSession):
         self.account_data = currentSession.get_current_config()
         self.download_formats = ('pdf', 'mobi', 'epub', 'code')
 
+    def get_ebook_data(self, element):
+
+        def ebook_title(element):
+            try:
+                text = element.find('div', {'class': 'title'}).getText()
+                return re.fullmatch('\s*(.+?)(\s*\[eBook\])?\s*', text, flags=re.IGNORECASE).group(1)
```

Comment:
There are some books with titles "Some stuff[eBook]" or "Some stuff [ebook]" - this regex should catch all the titles, including those invalid.